### PR TITLE
Change implementation of splash ("About") dialog

### PR DESCRIPTION
**Pull request by ejeschke**
- from channel viewer to a pop-up modal dialog
- shows version in title bar

### CODE REVIEW

**Comment by ejeschke** [comment]
@pllim, absolutely no hurry on this one. Just a small and cosmetic change.

File: ginga/rv/Control.py [hunk 1]
```diff
@@ -1543,25 +1543,38 @@ def scale2text(self, scalefactor):
...
-        header = image.get_header()
-        header['VERSION'] = __version__
+        w = Viewers.GingaViewerWidget(viewer=viewer)
+        w.resize(wd, ht)
+        vbox.add_widget(w, stretch=1)
 
-        if raiseTab:
-            self.change_channel(chname)
+        viewer.set_image(image)
+        top.show()
```
Comment (pllim) [inline ginga/rv/Control.py:1577]: Is there a way to suppress the pop up?

**Comment by pllim** [comment]
TBH, I never understood why splash banner was needed. In other commercial software, it only pops up when you go to Help and click About. Is that what this is doing?

**Comment by ejeschke** [comment]
It only pops up the first time you run the program and thereafter only when you select the "About" item from the Help menu.  It can (and is) suppressed by putting a "noBanner = True" in your $HOME/.ginga/general.cfg. It does this automatically the first time you run the program.

The only difference from the past is that in the past implementation it popped up as a "Ginga" channel tab, but with no mention of the version.

**Comment by pllim** [comment]
In that case, feel free to merge. Thanks!

**Comment by ejeschke** [comment]
Thanks, @pllim!

**Pull request merged**